docs/src/pages/components/pagination/TablePagination.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TablePagination from '@material-ui/core/TablePagination';
+import Paper from '@material-ui/core/Paper';
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 650,
+  },
+});
+
+function createData(color, rgb, hex) {
+  return { color, rgb, hex };
+}
+
+const rows = [
+  createData('Aqua', '0, 255, 255', '#00FFFF'),
+  createData('Blue', '0, 0, 255', '#0000FF'),
+  createData('Dodger Blue', '30, 144, 255', '#1E90FF'),
+  createData('Light Sky Blue', '135, 206, 250', '#87CEFA'),
+  createData('Navy', '0, 0, 128', '#000080'),
+  createData('Slate Blue', '106, 90, 205', '#6A5ACD'),
+  createData('Steel Blue', '70, 130, 180', '#4682B4'),
+  createData('Sky Blue', '135, 206, 235', '#87CEEB'),
+];
+
+export default function SimpleTable() {
```

Comment:
```suggestion
export default function TablePagination() {
```

Reply:
This creates a name conflict for me. Renamed `SimpleTablePagination`